docs/getting-started/first-autoscale.md

```diff
@@ -1,5 +1,5 @@
 # Scaling to Zero
-**Remember those super powers :rocket: we talked about?** One of Knative Serving's powers is automatic scaling or simply "autoscaling" out-of-the-box! This means your Knative Service will only "spin up" your application to perform its job (in this case, saying "Hello world!") if it is needed; otherwise, it will =="scale to zero"== by spinning down and waiting for a new request to come in.
+**Remember those super powers :rocket: we talked about?** One of Knative Serving's powers is automatic scaling or simply "autoscaling" out-of-the-box! This means your Knative Service will only "spin up" your application to perform its job (in this case, saying "Hello world!") if it is needed; otherwise, it will "scale to zero" by spinning down and waiting for a new request to come in.
```

Comment:
**Remember those super powers :rocket: we talked about?** One of Knative Serving's powers is built-in automatic scaling (autoscaling). This means your Knative Service only spins up your application to perform its job -- in this case, saying "Hello world!" -- if it is needed; otherwise, it scales to zero by spinning down and waiting for a new request to come in.

Reply:
Can you include these as suggestions next time so I can easily commit? 

http://haacked.com/archive/2019/06/03/suggested-changes/#:~:text=GitHub%20has%20a%20suggested%20Changes,the%20gutter%20on%20the%20left.

Reply:
Addressed everything but "scale to zero," it's a key word so I'd like to leave it in!